R/collections.R

```diff
@@ -49,4 +54,29 @@ irm <- function(x, trash = TRUE, recursive = FALSE, unregister = FALSE,
   # response
   invisible(out)
 }
+#' @rdname irm
```

Comment:
should be imkdir?

Reply:
I lumped `irm` and `imkdir` in one family of functions. This just says that I create 1 man-file for both functions.

Reply:
Oh - I see.

The naming is now confusing - perhaps there's a better name for the grouping.    Definitely not a blocker.

Reply:
Made some edits to the man. It would look now like this in Rstudio when calling `?irm` or `?imkdir`.

![imkdir_manpage](https://user-images.githubusercontent.com/24922632/198306382-abced8da-74f0-4153-9f34-9206101def4a.png)

I thougth this is a natural way of lumping those functions. What do you think?

Reply:
If grouping/lumping is desired over just referring to other commands directly, then I think it's okay.   I don't know what common practice is for grouping things like this in the R community.

Without that context/knowledge, I would have expected a page for `?irm` to refer to the page for `?imkdir` where that command would be explained, like entries in a dictionary (`see ?imkdir`).

Go with what feels natural for now I think.

Reply:
no, that's fine - i'll resolve this for now.   it's just docs - can tweak later if we change our minds.